cmake/schemes/url_sha1_odb-sqlite_autotools.cmake.in

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2013, 2015 Ruslan Baratov, Alexandre Pretyman
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.0)
+project(Hunter)
+
+include(ExternalProject) # ExternalProject_Add
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/modules")
+
+include(hunter_fatal_error)
+include(hunter_status_debug)
+include(hunter_test_string_not_empty)
+
+include(hunter_autotools_append_toolchain_binaries)
+include(hunter_autotools_append_preprocessor_flags)
+include(hunter_autotools_append_compiler_cxxflags)
+include(hunter_autotools_append_compiler_cflags)
+include(hunter_autotools_append_linker_flags)
+
+hunter_status_debug("Scheme: url_sha1_odb-sqlite_autotools")
+
+set(HUNTER_STATUS_DEBUG "@HUNTER_STATUS_DEBUG@")
+
+if("@MSVC@")
+  hunter_fatal_error("Autotools scheme not supported with Visual Studio")   
+endif()
+
+list(APPEND CMAKE_MODULE_PATH "@HUNTER_SELF@/cmake/find")
+
+set(ODB_ROOT "@ODB_ROOT@")
+find_package(odb REQUIRED)
+
+set(SQLITE3_ROOT "@SQLITE3_ROOT@")
+find_package(SQLite3 REQUIRED)
+
```

Comment:
quite frankly, it is already a hassle with 3 `url_sha1_*_autotools`. I was thinking of either a macro or function. Something like:

* hunter_autotools_init() - maybe with optional preprocessor flags, {c,cxx} compile flags and linker flags as parameters

I can't figure out what is better, a macro that sets the `configure_opts` with no scope restrictions or a function that sets `configure_opts` on `PARENT_SCOPE`, maybe a function is safer, right? What do you think?